can/parser_pyx.pyx

```diff
@@ -101,47 +102,34 @@ cdef class CANParser:
       message_options_v.push_back(mpo)
 
     self.can = new cpp_CANParser(bus, dbc_name, message_options_v, signal_options_v)
-    self.update_vl()
+    self.update_strings([])
 
-  cdef unordered_set[uint32_t] update_vl(self):
+  def update_string(self, dat, sendcan=False):
```

Comment:
let's get rid of the cython update_string. it's not used anywhere

Reply:
it's still used in can/tests/test_packer_parser.py and can/tests/test_parser.py.

should we replace it with `update_strings` in these two files? or keep update_string as it is?

Reply:
let's replace it